src/main/java/com/echobox/api/linkedin/util/JsonUtils.java

```diff
@@ -65,8 +65,22 @@ private static Object getValue(JsonValue jsonValue) {
       value = toMap(jsonValue.asObject());
     } else if (jsonValue.isString()) {
       value = jsonValue.asString();
+    } else if (jsonValue.isNumber()) {
+      value = getNumber(jsonValue);
     }
     return value;
   }
+  
+  private static Object getNumber(JsonValue jsonValue) {
+    try {
+      return jsonValue.asLong();
+    } catch (NumberFormatException e) {
```

Comment:
```suggestion
    } catch (NumberFormatException | UnsupportedOperationException e) {
```
Do we need to catch `UnsupportedOperationException `as well? or it should already be validated by `isNumber()`?